website/static/games/ridgerider_games/minesweeper/index.html

```diff
@@ -12,22 +12,22 @@
             <div id="board"></div>
```

Comment:
Add `<meta name="viewport" content="width=device-width, initial-scale=1">` to the head tag on line 8. This makes sure your pixel values scale correctly for different device types.  This will help you more than anything else.

More info: https://www.w3schools.com/css/css_rwd_viewport.asp